workflow/scripts/activity_only_features.R

```diff
@@ -44,14 +44,14 @@ abc <- abc %>%
     activity_base_squared = activity_base^2
   )
 
-# extract DNase-only feature columns and output names as vector
-if (snakemake@params$activity == "DNase") {
-  dnase_feat_cols <- config %>%
+# extract Activity-only feature columns and output names as vector
+if (snakemake@params$activity == "DHS") {
```

Comment:
we should document the "key" somewhere so people know to use DHS to indicate the DNase-only. assuming you changed it to be consistent with ABC?

Reply:
Yeah I made it consistent with ABC. Since we don't have any existing documentation, the next best thing to do is add an error message if "DHS" or "ATAC" isn't provided.

Reply:
oh 1 thing i should've probably asked you in your last PR would've been to update the README in the main page. There we can describe the new dataset_config file and the different params used